apps/omg_db/lib/omg_db/rocksdb/server.ex

```diff
@@ -91,17 +118,19 @@ defmodule OMG.DB.RocksDB.Server do
     {:reply, result, state}
   end
 
-  def handle_call(:utxos, _from, state) do
+  defp do_utxos(state) do
```

Comment:
hm, I just noticed that `measure_event()` is missing here. I don't see a reason why (it's missing also in `LevelDB` implementation).

@pthomalla was there a good reason to leave it out or just missed?

If missed, then let's add (in both impls) for consistency, when we have this cleanup PR at hand.

Reply:
yeah, indeed it's missing from ldb as well. @pthomalla please advise. I can add this in this PR no probs.

Reply:
Because there is no additional logic, to get_all_by_type 
but for consistency, we should add it here too.